PropertyTraits.h

```diff
@@ -145,6 +145,8 @@ namespace vive {
             vr::Prop_DriverDirectModeSendsVsyncEvents_Bool,
         DisplayDebugMode = vr::Prop_DisplayDebugMode_Bool,
         GraphicsAdapterLuid = vr::Prop_GraphicsAdapterLuid_Uint64,
+        DriverProvidedChaperonePath =
+            vr::Prop_DriverProvidedChaperonePath_String,

```

Comment:
Was this updated with the tool or manually? (Just wanted to make sure it was the tool so that it's complete)

Reply:
Yup, there were not a lot of changes there